src/components/InputBoxFreeType/FreeType.jsx

```diff
@@ -0,0 +1,26 @@
+import "./FreeType.scss";
+
+const FreeType = ({
```

Comment:
Please use camel case and add a dash between free-type
 and delete the last test for div